.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-yaml
+        args: [--unsafe]
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+  - repo: https://github.com/pycqa/isort
+    rev: 5.13.2  # Use the same version as in pyproject.toml
+    hooks:
+      - id: isort
+        args: ["--profile", "black", "--filter-files"]
+        name: isort (python)
+  - repo: https://github.com/ambv/black
```

Comment:
It looks that the https://github.com/psf/black-pre-commit-mirror_ is recommended. Can you update that please?

https://github.com/psf/black/blob/d1d4fc58d3b744db35ac832cddf1d076ba5f5e7f/.pre-commit-hooks.yaml#L1

Reply:
Done! 6e18f15